README.md

```diff
@@ -170,20 +170,22 @@ You may or may not need this, but if NT;RE crashes/segfaults on launch, then ins
 and set: `LD_PRELOAD=/usr/lib32/libtcmalloc.so %command%` as the launch argument.
 
 ## Server instructions
-1. To run a server, install "Source SDK Base 2013 Dedicated Server".
+1. To run a server, install "Source SDK Base 2013 Dedicated Server" (appid 243750).
```

Comment:
243750 is Source SDK Base 2013 Multiplayer according to steamdb, did you mean to have 244310 here?

Reply:
whoops good catch, yeah I mean 244310